notebooks/README.md

```diff
@@ -18,11 +14,9 @@ Use the links below to open any of these for interactive exploration in colab.
 
  - [XLA in Python][XLA_in_Python] - interactive exploration of the XLA compiler and computation model in python.
 
+Some of these notebooks were also recently moved into JAX's [online documentation](https://jax.readthedocs.io/en/latest/).
```

Comment:
nit: "Some additional notebooks were recently moved...", to make extra clear people should look at the docs :)